app/models/katello/upstream_pool.rb

```diff
@@ -4,10 +4,10 @@ class UpstreamPool < OpenStruct
 
     class << self
       def fetch_pools(params)
-        quantities_only = ::Foreman::Cast.to_bool(params.delete("quantities_only"))
+        quantities_only = ::Foreman::Cast.to_bool(params.delete(:quantities_only))
```

Comment:
I can change these back if there are objections but I do prefer to use symbols where we can